blocks/testing/test/qa_UI_Integration.cpp

```diff
@@ -11,6 +11,13 @@
 #include "gnuradio-4.0/testing/ImChartMonitor.hpp"
 #include "gnuradio-4.0/testing/TagMonitors.hpp"
 
+#ifdef _WIN32
+[[maybe_unused]] const auto _ = [] {
+    SetConsoleOutputCP(CP_UTF8);
+    return 0;
+}();
+#endif
+
```

Comment:
@chrisjohgorman I wonder whether this could be placed into the common [UnitTestHelper.hpp](https://github.com/fair-acc/gnuradio4/blob/main/meta/include/gnuradio-4.0/meta/UnitTestHelper.hpp) file and included in each unit test. This would avoid the platform-specific config duplication. 

Also, maybe add an `inline` to avoid ODR issues later.

What do you think?

*P.S. I restarted the clang CI check. I think this might be an unrelated flaky test.*

Reply:
@chrisjohgorman excellent! Your help and improvements are much appreciated!

Sorry for not getting back to you sooner. As you mentioned, the way you approach things is very good way to get acquainted with the code base. I'd wish/hope more will follow your example! :+1: